### PR TITLE
feat: add uutils pkg

### PR DESCRIPTION
**Pull request by tuananh**
Signed-off-by: Tuan Anh Tran <me@tuananh.org>

### CODE REVIEW

File: uutils.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://github.com/uutils/coreutils/archive/refs/tags/${{package.version}}.tar.gz
+      expected-sha256: a133449db283c145483e7945c925104007294d600b75991c5dad2cc91dc11d2e
+  - name: Configure and build
+    runs: |
+      cargo build --release --features unix
+      mkdir -p ${{targets.destdir}}/usr/local
+      make PREFIX=${{targets.destdir}}/usr/local install
```
Comment (kaniini) [inline uutils.yaml:32]: this should be PREFIX=${{targets.destdir}}/usr

**Review by rawlingsj — DISMISSED** [review]

**Comment by kaniini** [comment]
Is there a reason we aren't building uutils as a multicall?

**Comment by tuananh** [comment]
> Is there a reason we aren't building uutils as a multicall?

i wasn't aware of that option until now. this is to make it easy to replace busybox later on?

**Review by imjasonh — DISMISSED** [review]

**Comment by imjasonh** [comment]
CI ominously fails with

```
2023/02/17 19:27:43 melange (uutils/x86_64):   Downloaded time v0.3.17
2023/02/17 19:27:43 melange (uutils/x86_64): error: failed to verify the checksum of `time v0.3.17`
```

🤔

**Comment by tuananh** [comment]
looks like it's coming from rust. let me check it.

**Comment by tuananh** [comment]
@imjasonh works fine locally for me. not sure why it happens on CI.

**Review by imjasonh — APPROVED** [review]

**Pull request merged**